src/openapi.json

```diff
@@ -1349,20 +1349,20 @@
 				}
 			}
 		},
-		"/organizations/{organizationId}": {
+		"/changemakers/{changemakerId}": {
 			"get": {
-				"operationId": "getOrganizationById",
-				"summary": "Gets a specific organization.",
-				"tags": ["Organizations"],
+				"operationId": "getChangemakerById",
+				"summary": "Gets a specific changemaker.",
+				"tags": ["Changemakers"],
 				"security": [
 					{
 						"auth": []
 					}
 				],
 				"parameters": [
 					{
-						"name": "organizationId",
-						"description": "The PDC-generated ID of an organization.",
+						"name": "changemakerId",
+						"description": "The PDC-generated ID of an changemaker.",
```

Comment:
An changemaker or a changemaker?

Reply:
Asking the deep philosophical questions (thank you!  I did get a little "replace" happy in my manual find/replace)